velox/functions/sparksql/tests/ArithmeticTest.cpp

```diff
@@ -338,5 +338,15 @@ TEST_F(ArithmeticTest, hypot) {
   EXPECT_DOUBLE_EQ(5.70087712549569, hypot(3.5, -4.5).value());
 }
 
+TEST_F(ArithmeticTest, cot) {
+  const auto cot = [&](std::optional<double> a) {
+    return evaluateOnce<double>("cot(c0)", a);
+  };
+
+  EXPECT_EQ(cot(0), kInf);
+  EXPECT_EQ(cot(std::nullopt), std::nullopt);
```

Comment:
Remove tests for null inputs. These test the framework, not the function. Please, add some more tests.